licenses/templates/licenses/list.html

```diff
@@ -1,10 +1,42 @@
 {% extends 'base.html' %}
 
 {% load i18n %}
+{% load static %}
 
 {% block content %}
 <h1> {% translate 'Your licenses' %}</h1>
 
+<table class=table>
+    <thead>
+        <tr>
+            <th scope="col">{% translate 'License' %}</th>
+            <th scope="col">{% translate 'Confirmed' %}</th>
+        </tr>
+    </thead>
+    <tbody>
+        {% for license in object_list %}
+            <tr>
+                <td>
+                    <a href="{% url "licenses:details" license.id %}" class='link-dark'> {{license}} </a>
+                </td>
+                {% if license.confirmed %}
+                    <td>
+                    <img src={% static "admin/img/icon-yes.svg" %}
+                         alt={% translate 'Yes'%}>
```

Comment:
Interesting, that we have `"` for `href`  but not for src and alt. It seems that `"` is present for everything in the browser inspect mode but not in the source code. I would be happier if we have each attribute value enclosed with `"`, also for `class='link-dark'`.